src/test/java/org/cyclonedx/parsers/AbstractParserTest.java

```diff
@@ -549,11 +549,36 @@ void assertVulnerabilities(final Bom bom, final Version version) {
     assertEquals("https://example.com", vuln.getCredits().getOrganizations().get(0).getUrls().get(0));
 
     //Tools
-    assertEquals(1, vuln.getTools().size());
-    assertEquals("Sonatype CLI", vuln.getTools().get(0).getName());
-    assertEquals("Sonatype", vuln.getTools().get(0).getVendor());
-    assertEquals("1.131", vuln.getTools().get(0).getVersion());
-    assertEquals(1, vuln.getTools().get(0).getHashes().size());
+    if (version == Version.VERSION_15) {
+      assertNull(vuln.getTools());
+    }
+    else {
+      assertEquals(1, vuln.getTools().size());
+      assertEquals("Sonatype CLI", vuln.getTools().get(0).getName());
+      assertEquals("Sonatype", vuln.getTools().get(0).getVendor());
+      assertEquals("1.131", vuln.getTools().get(0).getVersion());
+      assertEquals(1, vuln.getTools().get(0).getHashes().size());
+    }
+
+    // ToolChoice
+    if (version == Version.VERSION_15) {
```

Comment:
this seems the same as the condition above, so I would merge both

Reply:
> this seems the same as the condition above, so I would merge both

👍 21abe5f0